src/app/lib/button/button-demo/button-demo.component.html

```diff
@@ -1,75 +1,83 @@
-    action is positve or in agreement. For negative actions use red. The purple and gray variations should be used only in
-    cases where more colors are needed or if they are being placed on backgrounds which green doesn’t contrast well enough.
-    Icons, if needed, should match text color of the button.</p>
-  <br>
-  <p class="description">
-    <b>Usage</b>
-    <br>Generally, there should only be one primary button per page. All other interactive items should either be secondary buttons
-    or text links. A few scenarios in which there would be multiple primary buttons: a modal with both an “accept” and “decline”
-    action, a page who’s content are non related and each content section have disparate actions associated with them. For
-    modals the preference is one primary button and a cancel action set as a text link. Below are some examples of buttons
-    usage.
-  </p>
+    <p class="description">
+        <b>Color Variations</b>
+        <br>Above are all acceptable colors and their active states. Green should be used for most cases and especially
+        cases the
+        action is positive or in agreement. For negative actions use red. The purple and gray variations should be used
+        only in
+        cases where more colors are needed or if they are being placed on backgrounds which green doesn’t contrast well
+        enough.
+        Icons, if needed, should match text color of the button.</p>
+    <br>
+    <p class="description">
+        <b>Usage</b>
+        <br>Generally, there should only be one primary button per page. All other interactive items should either be
+        secondary buttons
+        or text links. A few scenarios in which there would be multiple primary buttons: a modal with both an “accept”
+        and “decline”
+        action, a page who’s content are non related and each content section have disparate actions associated with
+        them. For
+        modals the preference is one primary button and a cancel action set as a text link. Below are some examples of
+        buttons
+        usage.
+    </p>
 
-  <br>
-  <hr style="height:1px;border:none;background-color:#E5E5E5;">
-  <br>
+    <br>
+    <hr style="height:1px;border:none;background-color:#E5E5E5;">
+    <br>
 
-  <div class="headerName">Angular Component</div>
-  <br>
-  <div id="HTML" *ngIf="showTemplate">
-    <div class="codeToggle">
-      <span class="toggleOptionOn" (click)="viewToggle('html')">HTML</span>
-      <span class="toggleOptionOff" (click)="viewToggle('ts')">TypeScript</span>
-    </div>
-    <pre><code>&#x3C;hc-button
+    <div class="headerName">Angular Component</div>
+    <br>
+    <div id="HTML" *ngIf="showTemplate">
+        <div class="codeToggle">
+            <span class="toggleOptionOn" (click)="viewToggle('html')">HTML</span>
+            <span class="toggleOptionOff" (click)="viewToggle('ts')">TypeScript</span>
+        </div>
+        <pre><code>&#x3C;hc-button
```

Comment:
this should be changed to show the proper usage of the button
ie instead of <hc-button> it should be <button hc-button